### PR TITLE
Add flakes support

### PR DESCRIPTION
**Pull request by tadeokondrak**
This makes it easy to use packages from here when using Nix flakes.
Feel free to close this if you don't/won't use flakes.

### CODE REVIEW

**Comment by sbruder** [comment]
Sorry for leaving this unanswered for so long, GitHub did not make me watch the repository so I did not get a notification about this PR.

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        import nixpkgs {
+          inherit system;
+          overlays = [ self.overlay ];
+        }
+      );
+    in
+    {
+      overlay = import ./default.nix;
+      packages = forAllSystems (system: {
+        inherit (nixpkgsFor.${system}) vapoursynthPlugins;
```
Comment (sbruder) [inline flake.nix:18]:
```suggestion
        inherit (nixpkgsFor.${system})
          getnative
          vapoursynthPlugins;
```
`getnative` is currently not accessible via the `packages` output

**Review by sbruder — CHANGES_REQUESTED** [review]
This looks nice, thank you! The only missing thing is `getnative` in the `packages` output (see inline comment).

**Comment by sbruder** [comment]
On another note, do you want to continue development upstream in your repository or in my fork?

I originally planned to open a PR to your repository but never found the time to test all plugins well enough (and e.g. debanding with placebo still produces strange artifacts that I couldn’t fix).

**Comment by tadeokondrak** [comment]
> On another note, do you want to continue development upstream in your repository or in my fork?
>
> I originally planned to open a PR to your repository but never found the time to test all plugins well enough (and e.g. debanding with placebo still produces strange artifacts that I couldn’t fix).

I don't use VapourSynth very often, but it would be nice to have a project like this that other people interested in VapourSynth and Nix can contribute to somewhere. It doesn't really matter to me if it's in this repo or mine.

I'll add an MIT license into my repo now. I'd definitely merge this work and make you a committer on the original repo if you'd be fine with that.

**Review by sbruder — APPROVED** [review]
Thanks for addressing my suggestions!

**Comment by sbruder** [comment]
> I'll add an MIT license into my repo now. I'd definitely merge this work and make you a committer on the original repo if you'd be fine with that.

Of course, thank you!

I’ll merge this later and try to test all plugins I added over the next few days so I can create a PR to your repository.

**Pull request merged**